### PR TITLE
Adding config and pubSub interfaces

### PR DESCRIPTION
**Pull request by davidjoy**
Updating logging interface to conform with the others.

Also removed old logging methods that we don’t want anymore.

### CODE REVIEW

File: src/config/configure.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+
+let service = null;
+
+export default function configure(ConfigService, services) {
+  service = new ConfigService(services);
+  PropTypes.checkPropTypes({
```
Comment (abutterworth) [inline src/config/configure.js]: Is this working?

File: src/logging/NewRelicLoggingService.js [hunk 2]
```diff
@@ -28,98 +29,22 @@ class NewRelicLoggingService {
     }
   }
 
-  static logError(error, customAttributes) {
+  logError(error, customAttributes = {}) {
+    const errorCustomAttributes =
+      error.customAttributes !== undefined ? error.customAttributes : {};
```
Comment (abutterworth) [inline src/logging/NewRelicLoggingService.js]: Would `const errorCustomAttributes = error.customAttributes || {}` be safer?

File: src/config/configure.js (hunk 1 above)
Comment (davidjoy) [reply src/config/configure.js]: Wow.  Uh.  Sure do.  I failed to finish that code. ;)  Good catch.

File: src/logging/configure.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+
+let service = null;
+
+export function configure(LoggingService, services) {
+  service = new LoggingService(services);
+
+  PropTypes.checkPropTypes({
```
Comment (abutterworth) [inline src/logging/configure.js]: Same thing here: is this a correct usage of checkPropTypes?

File: src/config/configure.js (hunk 1 above)
Comment (abutterworth) [reply src/config/configure.js]: Oh interesting you're using this to validate the implementation. Neat.

File: src/pubSub/PubSubJsService.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+import PubSub from 'pubsub-js';
+
+export default class PubSubJsService {
```
Comment (abutterworth) [inline src/pubSub/PubSubJsService.js]: What's the need for this?

**Pull request merged**